### ISSUE TITLE
🐞 Bug buildessential metapackage fails on armv7l userspace: linuxheaders not found

### ISSUE THREAD

**Issue by blu** [chromebrew/chromebrew#9312]
**Describe the bug**

buildessential metapackage fails on armv7l userspace: linuxheaders not found

**To Reproduce**

1. I executed this command:

      ```shell
      $ crew install buildessential
      ```

      Output:

      ```
      linuxheaders: Linux headers for Chrome OS.
      Precompiled binary available, downloading...
      Download failed with error 404: Not Found
      ```

**Environment: Paste the output of `crew sysinfo -v` between the two `***` below**

***
<details><summary>Expand</summary>

- Architecture: `aarch64` (`armv7l`)
- Processor vendor: `ARM`
- User space: `32-bit`
- Chromebrew Kernel version: `5.15`
- Chromebrew Running in Container: `false`

- Chromebrew version: `1.43.2`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib`

- Last update in local repository: 66b0446a1 `Remove a number of unnecessary instance variables (#9310) (4 hours ago)`

- OS variant: `Chrome OS`
- OS version: `hana-release/R120-15662.88.0`
- OS channel: `stable-channel`

</details>
Crew is exiting.
***

**Comment by uberhacker** [comment]
@blu: I'm not able to repeat this.  Maybe try to upgrade to release 121?  See https://www.thegeekstuff.com/2017/08/chrome-os-update-to-latest-version/.

**Comment by blu** [comment]
@uberhacker unfortunately I can't update the machine in question, as the next stable-channel update bricks the device to a point of needing restoration. The above issue with linuxheaders is on a freshly restored device,

**Comment by Zopolis4** [comment]
What's the output of `crew const CREW_KERNEL_VERSION`? I think I might know what the problem is.

**Comment by satmandu** [comment]
Running `uname -a` would give us the info we need...

**Comment by Zopolis4** [comment]
I think the problem is that binaries for kernel version `4.14` are uploaded under version `4.14-1`, which means the binary url generation is broken-- we can just probably reupload the existing binaries

**Comment by blu** [comment]
> What's the output of `crew const CREW_KERNEL_VERSION`? I think I might know what the problem is.

`CREW_KERNEL_VERSION` is the same as the one given by `crew sysinfo -v` -- 5.15. Here's the complete version info as given by`uname`:
```
Linux localhost 5.15.137-20865-g0a54da9da41a #1 SMP PREEMPT Thu Jan 25 19:26:58 PST 2024 aarch64 GNU/Linux
```

**Comment by blu** [comment]
BTW, on a tangent re package `linuxheades`: one cannot `crew reinstall -s linuxheaders` without an already installed linuxheaders, as the build expects headers found in the package.

**Comment by Zopolis4** [comment]
Oh-- the problem is that we simply do not have a 5.15 linuxheaders package for armv7l.

https://github.com/chromebrew/chromebrew/blob/14d23831378db63c5d7bbfa4d81f82b6ba5756f5/packages/linuxheaders.rb#L53-L70

https://github.com/chromebrew/chromebrew/blob/master/packages/linuxheaders.rb#L42-L54

The hashes of the supposed armv7l 5.15 are actually just the 5.10 headers.

**Comment by blu** [comment]
BTW, commit 3c1dba2 fixes the availability of 5.15 linuxheaders binaries, but it seems commit uses the wrong binary_sha256 from 5.10, so `crew install` fails with a checksum mismatch.

**Comment by blu** [comment]
Ok, I missed @satmandu PR above. That should resolve all 5.15 linuxheaders binaries.
